data/eos-demo-video.desktop.in.in

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+_Name=Demo Video
+Exec=@LIBEXECDIR@/eos-demo-video
+Terminal=false
+Categories=
+Type=Application
+NoDisplay=true
+AutostartCondition=if-exists /home/demo-guest
```

Comment:
As mentioned elsewhere, this is not a great condition for this. What we really want to check for is "is the user that just logged in the demo guest", not "does the demo guest user home directory exist".
A marker file sounds like a better option, but a slight corner case is that unless you remove the file after login, it would probably check for "is the demo guest user logged in" instead. (I think it'd still be OK.)